src/components/radio/Radio.js

```diff
@@ -178,7 +192,7 @@ export default class RadioComponent extends Field {
       && this.previousValue === this.currentValue;
     if (shouldResetValue) {
       this.resetValue();
-      this.triggerChange(flags);
+      this.triggerChange();
```

Comment:
Why was this changed? This will certainly introduce a bug somewhere if we accept this. We should be passing along the flags since they are commonly used to flag behaviors during the onChange event loop.

Reply:
fixed (working from 4.12.4)